.gitignore

```diff
@@ -1,5 +1,6 @@
 .env
 .vscode/
+.idea
```

Comment:
This change seems unrelated to the issue this PR solves. It's better to remove it